components/ws-proxy/pkg/sshproxy/server.go

```diff
@@ -318,29 +318,30 @@ func (s *Server) HandleConn(c net.Conn) {
 		OwnerUserId: wsInfo.OwnerUserId,
 	}
 
-	if wsInfo.SSHKey != nil {
```

Comment:
question, blocking:
@iQQBot what scenario is `wsInfo.SSHKey != nil` for? Is it still needed? If no longer needed, can we remove in this PR too? Asking because if we don't do now, it might "linger".

Reply:
Should we wait to land this PR until Monday? If we land it today, it could ship Monday. This would not be ideal, because pods config could apply after pods, falling back to ssh host.

When this PR lands, do we also need a ws-manager-mk2 change to land? Otherwise ws-manager-mk2 has the same risk.

Reply:
Was nextgen the only one using wsInfo.SSHKey?